### ISSUE TITLE
Unable to connect to 'criserver' using 'crictl'

### ISSUE THREAD

**Issue by gabhijit** [cri-o/cri#201]
1. I am able to build the project successfully after running `cargo build` with latest stable rust (rustc - v1.47.0)

2. Then I run 

```
$ ./target/debug/criserver -l trace
[2020-10-30T08:02:18.484Z TRACE cri::storage::default_key_value_storage src/storage/default_key_value_storage.rs:22] Opening storage /var/run/user/1000/cri/storage/cri-service
[2020-10-30T08:02:18.605Z DEBUG cri::network::cni src/network/cni/mod.rs:126] Configuration paths: /etc/cni/net.d
[2020-10-30T08:02:18.605Z DEBUG cri::network::cni src/network/cni/mod.rs:135] Plugin paths: <some-bin-paths>
[2020-10-30T08:02:18.605Z INFO  cri::network::cni src/network/cni/mod.rs:139] Initializing CNI network
[2020-10-30T08:02:18.605Z INFO  cri::network::cni src/network/cni/mod.rs:141] No default CNI network name, choosing first one
[2020-10-30T08:02:18.605Z DEBUG cri::network::cni src/network/cni/mod.rs:264] Got network files: []
[2020-10-30T08:02:18.605Z INFO  cri::network::cni src/network/cni/mod.rs:291] No loaded networks
[2020-10-30T08:02:18.606Z TRACE cri::network::cni src/network/cni/mod.rs:150] Setup CNI storage in /var/run/user/1000/cri/storage/cni
[2020-10-30T08:02:18.606Z TRACE cri::storage::default_key_value_storage src/storage/default_key_value_storage.rs:22] Opening storage /var/run/user/1000/cri/storage/cni
[2020-10-30T08:02:18.608Z INFO  cri::kubernetes::server src/kubernetes/server/mod.rs:65] Runtime server listening on /var/run/user/1000/cri/cri.sock
```

3. Trying to connect to it using 'crictl'  ( version 1.17.0) 

```
$ crictl -D -r "unix:///var/run/user/1000/cri/cri.sock.orig" version
DEBU[0000] VersionRequest: &VersionRequest{Version:v1alpha2,} 
DEBU[0000] VersionResponse: nil                         
FATA[0000] getting the runtime version failed: rpc error: code = Internal desc = stream terminated by RST_STREAM with error code: PROTOCOL_ERROR 
```

Not sure if I am missing something?

**Comment by saschagrunert** [comment]
Hey @gabhijit :wave:, I think this is a duplicate of https://github.com/cri-o/cri/issues/34. The main issue resides in h2, and there is already a PR open to fix that.

**Comment by gabhijit** [comment]
Yes. Indeed it looks like. Should this be closed in that case if it's duplicate of #34

**Comment by saschagrunert** [comment]
Thank you! I'll just gave my h2 PR a bump
/close
